format/binary.md

```diff
@@ -439,7 +443,7 @@ When interleaved, the same array would instead look like this:
 |:-----:|:--------:|:--------:|:--------:|:--------:|
 | Row 1 | `A0`     | `B0`     | `C0`     | `D0`     |
 | Row 2 | `A1`     | `B1`     | `C1`     | `D1`     |
-| Row 3 | `A2`     | `B2`     | `C2`     | `D2`     |
+p| Row 3 | `A2`     | `B2`     | `C2`     | `D2`     |
```

Comment:
That `p` shouldn't be there

Reply:
I had to fix this in the other pr too oof x_x

I’ll get on it shortly :)